docs/installation/manage/_index.md

```diff
@@ -109,50 +141,71 @@ brew services restart viam-server
 {{% /tab %}}
 {{< /tabs >}}
 
-## Troubleshooting
+## View `viam-server` logs

```

Comment:
Moved troubleshooting content out to Troubleshooting, as it was not manage-specific, but also helps with installation.